### ISSUE TITLE
Tablesort doesn't work properly when instant loading is enabled

### ISSUE THREAD

**Issue by fveauvy** [squidfunk/mkdocs-material#1874]
__I checked that...__

- [X] ... the documentation does not mention anything about my problem
- [X] ... the problem doesn't occur with the default MkDocs template
- [X] ... the problem is not in any of my customizations (CSS, JS, template)
- [X] ... there are no open or closed issues that are related to my problem

## Description

Data tables aren't sortable when instant loading feature is enabled.

### Expected behavior

I followed the guidelines from the [docs](https://squidfunk.github.io/mkdocs-material/reference/data-tables/#sortable-tables) in order to make data tables sortable with the instant loading feature.

### Actual behavior

The data tables aren't sortable when we access the page. I need to manually refresh my web navigator to make them sortable.
I noticed that when I leave a page, the data table seems to be sortable for a brief instant before the new content appears.

### Steps to reproduce the bug

1. Created a new mkdocs doc with ` mkdocs new . `
2. Declared the material theme, enabled instant feature and added the extra JS needed for tableSort
3. `mkdocs serve`

### Package versions

* Python: 3.8.5
* MkDocs: 1.1.2
* Material: 5.5.7+insiders.1.2.0

### Project configuration

``` yaml
site_name: My Docs
theme:
  name: material
  features:
    - instant

extra_javascript:
  - https://cdnjs.cloudflare.com/ajax/libs/tablesort/5.2.1/tablesort.min.js
  - javascripts/tables.js
```

### System information

I tried with two different environments :

* OS: macOS Catalina
* Browser: Google Chrome / Firefox

* OS: Windows Subsystem for Linux (Ubuntu 20.04)
* Browser: Google Chrome

**Comment by squidfunk** [comment]
Thanks for reporting! The documentation was indeed wrong – it was the wrong observable to subscribe to. If you change `app.location$` to `app.document$`, my testing shows it works 😊 I corrected the documentation in 3d1491e6.

**Comment by fveauvy** [comment]
Sweet ! Thanks for the swift reply.
I'll close this issue.
